File: solutions/elixir/134-move-zeroes.ex
# Problem 134: Move Zeroes (LeetCode 283)
# Difficulty: Easy
# Language: Elixir
# 
defmodule Solution do
  def move_zeroes(nums) do
    {zeros, non_zeros} = Enum.split_with(nums, &(&1 == 0))
    non_zeros ++ zeros
  end
end

File: solutions/clojure/134-move-zeroes.clj
;; Problem 134: Move Zeroes (LeetCode 283)
;; Difficulty: Easy
;; Language: Clojure
;; 
(defn move-zeroes [nums]
  (let [non-zeros (filter #(not= 0 %) nums)
        zeros (filter #(= 0 %) nums)]
    (concat non-zeros zeros)))
